rpcserver.go

```diff
@@ -1832,6 +1838,53 @@ func GetUnconfirmedBalance(w *wallet.Wallet, chainSvr *chain.Client, icmd btcjso
 	return (unconfirmed - confirmed).ToBTC(), nil
 }
 
+// ImportAddress handles an importaddress request by parsing
+// a encoded address and importing it as a watch-only address to the
+// imported address account.
+func ImportAddress(w *wallet.Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interface{}, error) {
+	cmd := icmd.(*btcjson.ImportAddressCmd)
+
+	addr, err := btcutil.DecodeAddress(cmd.Address, activeNet.Params)
+	if err != nil {
+		return nil, btcjson.Error{
+			Code:    btcjson.ErrInvalidAddressOrKey.Code,
+			Message: err.Error(),
+		}
+	}
+
+	if !addr.IsForNet(activeNet.Params) {
+		return nil, btcjson.ErrInvalidAddressOrKey
+	}
+
+	_, err = w.ImportAddress(addr, nil, cmd.Rescan)
+	return nil, err
+}
+
+// ImportPubKey handles an importpubkey request by parsing
+// a raw public key and importing it as a watch-only address to the
```

Comment:
OK, updated the doc. Also, we're writing compressed/uncompressed based on the input pubkey now.